erlang/README.md

```diff
@@ -17,13 +17,17 @@ using apt:
 
 You need Erlang Client binaries:
 
-    wget http://www.rabbitmq.com/releases/rabbitmq-erlang-client/v2.7.0/rabbit_common-2.7.0.ez
-    unzip rabbit_common-2.7.0.ez
-    ln -s rabbit_common-2.7.0 rabbit_common
+    wget https://www.rabbitmq.com/releases/rabbitmq-erlang-client/v3.6.14/rabbit_common-3.6.14.ez
```

Comment:
Please switch to downloading from GitHub. rabbitmq.com/releases is not going to be around forever. No reason to not use 3.7.2 either.